src/GovUk.Education.ExploreEducationStatistics.Publisher/Events/ReleaseVersionPublishedEvent.cs

```diff
@@ -1,21 +1,36 @@
-﻿using System;
+using System;
 using Azure.Messaging.EventGrid;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.EventGrid;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Services;
 
 namespace GovUk.Education.ExploreEducationStatistics.Publisher.Events;
 
-public record ReleaseVersionPublishedEvent(Guid ReleaseVersionId, ReleaseVersionPublishedEvent.EventPayload Payload)
+public record ReleaseVersionPublishedEvent : IEvent
 {
+    public ReleaseVersionPublishedEvent(PublishingCompletionService.PublishedReleaseVersionInfo publishedReleaseVersion)
+    {
+        Subject = publishedReleaseVersion.ReleaseVersionId.ToString();
+        Payload = new()
+        {
+            ReleaseId = publishedReleaseVersion.ReleaseId,
+            ReleaseSlug = publishedReleaseVersion.ReleaseSlug,
+            PublicationId = publishedReleaseVersion.PublicationId,
+            PublicationSlug = publishedReleaseVersion.PublicationSlug,
+            PublicationLatestPublishedReleaseVersionId = publishedReleaseVersion.PublicationLatestPublishedReleaseVersionId,
+        };
+    }
+
     // Changes to this event should also increment the version accordingly.
     public const string DataVersion = "1.0";
     public const string EventType = "release-version-published";
     
     // Which Topic endpoint to use from the appsettings
-    public const string EventTopicOptionsKey = "ReleaseVersionChangedEvent";
+    public static string EventTopicOptionsKey => "ReleaseVersionChangedEvent";
     
     /// <summary>
     /// The ReleaseVersionId is the subject
     /// </summary>
-    public string Subject => ReleaseVersionId.ToString();
+    public string Subject { get; init; }
```

Comment:
`get; init;` isn't consistent with the other event records which only use `get;` for their accessors